### DISCUSSION TITLE
Alternative sequence creation method

### DISCUSSION THREAD

**Discussion by mrbelowski** [helgoboss/helgobox#1259] [Ideas]
I use Playtime when recording / arranging and often find myself wanting to shortcut the process of making a sequence. It would be nice to be able to specify the rows / elements / repeats manually ("2x row 1, then 1x row 2, then 4x row 3, etc etc) without having to record the sequence 'live'.

Doesn't need to be a shiny user interface - just some simple way to modify (or create) a sequence would be enough

**Comment by helgoboss** [comment]
You can select the scene and then press A to export it directly to the arrangement. The cursor automatically moves, so if you do this again, the scene will exported to the next position.

If you want to decide how long one scene should play, you can create a time selection in the REAPER project before doing the export.
